vmdb/app/assets/javascripts/cfme_application.js

```diff
@@ -243,8 +243,12 @@ function miqDimDiv(divname, status) {
 // Check for changes and prompt
 function miqCheckForChanges() {
   if(cfmeAngularApplication.$scope) {
-    if (cfmeAngularApplication.$scope.form.$dirty)
-      return confirm("Abandon changes?");
+    if (cfmeAngularApplication.$scope.form.$dirty) {
+      answer = confirm("Abandon changes?");
+      if(answer == true)
+        cfmeAngularApplication.$scope.form.$setPristine(true);
+      return answer;
+    }
```

Comment:
Can we not use "cfme" in upstream?  I just noticed this file today when dealing with  a BZ ticket I have, and I was surprised considering this is new code.